.github/workflows/main.yml

```diff
@@ -2,13 +2,13 @@ name: Run Examples
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
```

Comment:
Maybe we should do that thing where we include both master and main here for the smoothest transition?

Reply:
Actually, I just had a thought about this. When we migrate the branches, the only one that will be protected is main and then we could have the reverse issue as in the pytorch/pytorch. We either need to protect both, or just leave this be and merge/change the branch name together

Reply:
I think it's not a big deal on this repo so I could go either way, but I think it might just be easiest to just keep it as is and do the swap quickly.